sample-notificator-app/src/main/groovy/life/qbic/samplenotificator/datasource/notification/create/FetchSubscriberDbConnector.groovy

```diff
@@ -60,7 +61,35 @@ class FetchSubscriberDbConnector implements FetchSubscriberDataSource{
         }
         return foundNotifications
     }
+    
+    @Override
+    Map<String, String> fetchProjectsWithTitles() {
+
+      Map<String, String> projectsWithTitles = new HashMap<>()
+        try{
+            Connection connection = connectionProvider.connect()
+
+            connection.withCloseable { Connection con ->
+                    String sqlQuery = "select openbis_project_identifier, short_title from projects"
+
+                    PreparedStatement preparedStatement = con.prepareStatement(sqlQuery)
+                    preparedStatement.execute()
+                    def resultSet = preparedStatement.getResultSet()
 
+                    while(resultSet.next()){
+                        String identifier = resultSet.getString("openbis_project_identifier")
+                        String code = identifier.substring(identifier.length() - 5)
```

Comment:
why -5?

Reply:
bzw how does an identifier look like?

Reply:
identifier = /space/projectCode (which is 5 symbols long)
It's the last 5 symbols, didn't want to perform splits, which might lead to array out of bounds exceptions